kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt

```diff
@@ -451,6 +455,30 @@ interface ViewPagerAssertions : BaseAssertions {
 }
 
 /**
+ * Provides assertios for RatingBar
+ */
+interface RatingBarAssertions : BaseAssertions {
+    /**
+     *  Checks If rating bar is have number of rating as expected
+     *  @param number rating as expected
+     */
+    fun expectedNumberOfStarAs(number: Float) {
```

Comment:
To make the library consistent, the name should be `fun hasRating(rating: Float)`